src/fun/builtins.bend

```diff
@@ -175,3 +175,120 @@ hvm to_u24:
 # Casts any native number to an i24.
 hvm to_i24:
   ($([i24] ret) ret)
+
+# Encoding
+
+Utf8/MAX_RUNE = '\u{0010FFFF}'
+Utf8/RUNE_ERROR = '\u{FFFD}'
+#Utf8/RUNE_SELF = 0x80
+#Utf8/MAX = 4
+
+Utf8/t1 = 0b00000000
+Utf8/tx = 0b10000000
+Utf8/t2 = 0b11000000
+Utf8/t3 = 0b11100000
+Utf8/t4 = 0b11110000
+
+Utf8/maskx = 0b00111111
+Utf8/mask2 = 0b00011111
+Utf8/mask3 = 0b00001111
+Utf8/mask4 = 0b00000111
+
+Utf8/rune1max = (- (<< 1 7) 1)
+Utf8/rune2max = (- (<< 1 11) 1)
+Utf8/rune3max = (- (<< 1 16) 1)
```

Comment:
Do these need to be functions? Do we expect to use them in other places?
If no then we can make them into `let` or `use` terms.